assets/js/generator.js

```diff
@@ -94,11 +97,11 @@ function populateGeneratorList(data) {
         }
     }
 
-    var generatorArray = [];
-    $.each(generators, function (generatorLang, lang) {
+    var generatorArray /*: string[][] */ = [];
```

Comment:
Suggest using tuple type here and the equivalent place in `generator.js`.

Reply:
Since JavaScript has no native tuples, I assume you're referring to the annotations only?